samanthas_telegram_bot/api_queries.py

```diff
@@ -0,0 +1,91 @@
+# TODO for now this module only contains dummy functions for the conversation flow to work.
+import csv
+import logging
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+
+async def chat_id_is_registered(chat_id: int) -> bool:
+    """Checks whether the chat ID is already stored in the database."""
+    logger.info(f"Checking with the backend if chat ID {chat_id} exists...")
+    return False
+
+
+async def get_age_ranges() -> dict[str, list[dict[str, int]]]:
+    logger.info("Getting age ranges from the backend...")
+    return {
+        "student": [
+            {"age_from": 5, "age_to": 6},
+            {"age_from": 7, "age_to": 8},
+            {"age_from": 9, "age_to": 10},
+            {"age_from": 11, "age_to": 12},
+            {"age_from": 13, "age_to": 14},
+            {"age_from": 15, "age_to": 17},
+            {"age_from": 18, "age_to": 20},
+            {"age_from": 21, "age_to": 25},
+            {"age_from": 26, "age_to": 30},
+            {"age_from": 31, "age_to": 35},
+            {"age_from": 36, "age_to": 40},
+            {"age_from": 41, "age_to": 45},
+            {"age_from": 46, "age_to": 50},
+            {"age_from": 51, "age_to": 55},
+            {"age_from": 56, "age_to": 60},
+            {"age_from": 61, "age_to": 65},
+            {"age_from": 66, "age_to": 70},
+            {"age_from": 71, "age_to": 75},
+            {"age_from": 76, "age_to": 80},
+            {"age_from": 81, "age_to": 85},
+            {"age_from": 86, "age_to": 90},
+            {"age_from": 91, "age_to": 95},
+        ],
+        "teacher": [],
+        "matching": [],
+    }
+
+
+def get_assessment_questions(lang_code: str) -> tuple[dict[str, str], ...]:
+    """Gets assessment questions, based on language and level"""
+
+    # for some strange reason another ".parent" doesn't work, but ".." does
+    DATA_DIR = Path(__name__).parent / ".." / "data"
```

Comment:
`parent.parent` should work instead of `.parent / ".."`

Reply:
I know it should but it didn't 🙄 
And I will be getting the questions from the backend anyway